plugins/devtools/src/components/Content/ScheduledTasksContent/ScheduledTasksContent.tsx

```diff
@@ -0,0 +1,279 @@
+        const status = rowData.taskState?.status;
+
+        if (status === 'idle') {
+          return (
+            <StatusDisplay icon={<NightsStay fontSize="small" />} text="Idle" />
+          );
+        }
+
+        if (status === 'running') {
+          return (
+            <StatusDisplay
+              icon={<CircularProgress color="inherit" size="30px" />}
+              text="Running"
+            />
+          );
+        }
+
+        return <Typography variant="body2">{status || 'N/A'}</Typography>;
+      },
+    },
+    {
+      title: 'Last Run',
+      field: 'taskState.lastRunEndedAt',
+      width: '25%',
+      render: (rowData: TaskApiTasksResponse) =>
+        rowData.taskState?.lastRunEndedAt
+          ? new Date(rowData.taskState.lastRunEndedAt).toLocaleString()
+          : 'N/A',
+    },
+    {
+      title: 'Next Run',
+      width: '15%',
+      render: (rowData: TaskApiTasksResponse) =>
+        rowData.taskState?.status === 'idle' && rowData.taskState.startsAt
+          ? new Date(rowData.taskState.startsAt).toLocaleString()
+          : 'N/A',
+    },
+    {
+      title: 'Actions',
+      render: (rowData: TaskApiTasksResponse) => (
+        <RequirePermission permission={devToolsTaskSchedulerCreatePermission}>
+          <Tooltip title="Refresh">
+            <IconButton
+              aria-label="Trigger"
+              onClick={() => {
+                triggerTask(selectedPlugin, rowData.taskId);
+                if (isTriggering) {
+                  <CircularProgress color="inherit" size="30px" />;
+                }
+                if (triggerError) {
+                  alertApi.post({
+                    message: `Error triggering task ${rowData.taskId}: ${error}`,
+                    severity: 'error',
+                  });
+                } else {
+                  alertApi.post({
+                    message: `Successfully triggered task ${rowData.taskId}`,
+                    severity: 'success',
+                  });
+                }
+              }}
+            >
+              <RefreshIcon />
+            </IconButton>
+          </Tooltip>
+        </RequirePermission>
+      ),
+      sorting: false,
+      width: '10%',
+    },
+  ];
+
+  return (
+    <Box>
+      <Autocomplete
+        className={classes.formControl}
+        classes={{ root: classes.formControl }}
+        freeSolo
+        options={plugins}
+        value={selectedPlugin}
+        inputValue={inputValue}
+        onChange={handleAutocompleteChange}
+        onInputChange={(_event, newInputValue) => {
+          setInputValue(newInputValue);
+        }}
+        renderInput={params => (
+          <TextField
+            {...params}
+            label="Select Plugin"
+            variant="outlined"
+            onKeyDown={handleKeyDown}
+            onBlur={handleCommitChange}
+          />
+        )}
+      />
+
+      {loading && <Progress />}
+
+      {error && <Alert severity="error">{error}</Alert>}
```

Comment:
I wonder if we can handle this a bit nicer? When I use `techdocs` I get this:

<img width="385" height="174" alt="Image" src="https://github.com/user-attachments/assets/74f8b0ce-0d24-4724-b8f5-3b34fc9e4612" />

Having an empty state like component with some helpful details like "The plugin id provided doesn't have any scheduled task or may have a typo, please correct"